docs/01-command-line-reference.md

```diff
@@ -200,6 +200,33 @@ export default commandLineArgs => {
 }
 ```
 
+Config Intellisense
```

Comment:
You should make this a `####` sub-heading so that it looks right on the web site

Reply:
Otherwise looks great!

Reply:
added: https://github.com/rollup/rollup/pull/4127/commits/555cac31aa8cbf61c3a17b4c47e33bde720bfb5a